rskj-core/src/main/java/co/rsk/rpc/netty/RskWebSocketJsonRpcHandler.java

```diff
@@ -48,25 +52,25 @@
  */
 
 @Sharable
-public class RskJsonRpcHandler
+public class RskWebSocketJsonRpcHandler
         extends SimpleChannelInboundHandler<ByteBufHolder>
         implements RskJsonRpcRequestVisitor {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RskJsonRpcHandler.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(RskWebSocketJsonRpcHandler.class);
 
     private final EthSubscriptionNotificationEmitter emitter;
     private final JsonRpcSerializer serializer;
 
-    public RskJsonRpcHandler(EthSubscriptionNotificationEmitter emitter, JsonRpcSerializer serializer) {
+    public RskWebSocketJsonRpcHandler(EthSubscriptionNotificationEmitter emitter, JsonRpcSerializer serializer) {
         this.emitter = emitter;
         this.serializer = serializer;
     }
 
     @Override
-    protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder msg) {
-        try {
-            RskJsonRpcRequest request = serializer.deserializeRequest(
-                    new ByteBufInputStream(msg.copy().content())
-            );
+    protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder msg) throws IOException {
```

Comment:
I THINK that this `throws` keyword is not needed here since we're handling that inside the try catch and AFAIK the `content.release...` does not throws this exception.

Reply:
removed